### PR TITLE
Only print visible layers

### PR DESCRIPTION
**Pull request by LukasLohoff**
See title. @terrestris/devs Please review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terrestris/ol-util",
-  "version": "7.0.0",
+  "version": "7.1.0",
```
Comment (dnlkoch) [inline package.json:3]: Is this change intended?

Comment (LukasLohoff) [reply package.json]: no, somehow the commit from the last release made it into this MR. fixed via rebase

**Review by annarieger — APPROVED** [review]
